src/transformStateWithClones.js

```diff
@@ -66,6 +66,43 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+  let lastClone = { ...state };
+  const cloneStates = [];
+
+  for (const key of transforms) {
+    switch (key.operation) {
```

Comment:
Я бы тут использовал деструктуризацию, чтобы не писать key.operation и key.properties
```suggestion
  for (const { operation, properties } of transforms) {
    switch (operation) {
```